pom.xml

```diff
@@ -4,24 +4,22 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>HelloSwirld</groupId>
+    <groupId>hello.world</groupId>
     <artifactId>HelloSwirld</artifactId>
     <version>1.0-SNAPSHOT</version>
 
     <properties>
-        <sdk.dir>/home/swirlds/sdk</sdk.dir>
-        <app.name>HelloSwirld</app.name>
-
-        <app.repo>${sdk.dir}/data/repos/app/</app.repo>
-        <platform.repo>${sdk.dir}/data/repos/platform/</platform.repo>
+        <swirlds.sdk>/users/connor/Documents/hashgraph/sdk</swirlds.sdk>
```

Comment:
SDK location probably leaked from your local setup. I propose some generic unix path (or revert to "/home/swirlds/sdk") if nothing else comes to mind.